view/script/trial.js

```diff
@@ -17,6 +16,38 @@
         }]
     };
     var ctx = document.getElementById('trialChart').getContext('2d');
+    var differenceValue = 0;
+
+    document.getElementById('toDate').setAttribute('min', document.getElementById('fromDate').value);
+    $('#fromDate').on('change', function changeToDateValue () {
+        var fromDateValue = document.getElementById('fromDate').value;
+        var newtoDateValue = moment(fromDateValue).add(differenceValue, 'days').format('YYYY-MM-DD');
+
+        document.getElementById('toDate').value = newtoDateValue;
+    }).on('focus', function storeDateDiff (event) {
+        var oldfromDateValue = event.target.value;
+        var oldtoDateValue = document.getElementById('toDate').value;
+
+        differenceValue = moment(oldtoDateValue).diff(moment(oldfromDateValue), 'days');
+    });
+
+    $('#check-compliance').submit(function submitAction (event) {
+        event.preventDefault();
+        $.ajax({
+            type: 'GET',
+            data: $('#check-compliance').serialize() + '&requestType=ajaxRequest'
+        })
+        .done(function successSubmit (newData) {
```

Comment:
so since it is a named function it could be moved to an upper scope.

See http://eslint.org/docs/rules/max-nested-callbacks example for how that would look